### PR TITLE
Gz crypto list

### PR DESCRIPTION
**Pull request by gzucca**
- Home component "Main" created with empty component and button to toggle loading "CryptCards".
- CryptCards component created (will then be the one connecting with the API).
- CryptCard component created (hard-coded until connection with API).

### CODE REVIEW

**Review by FedericoLeiva12 — CHANGES_REQUESTED** [review]

File: src/components/CryptCard.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import {StyleSheet, Text, View, Image} from 'react-native';
+import React from 'react';
+import crypto from '../../metrics.json';
+import GainArrow from '../../assets/icons/north_east_black_24dp.svg';
+import LossArrow from '../../assets/icons/south_west_black_24dp.svg';
+
+type Props = {};
```
Comment (tomasdematos) [inline src/components/CryptCard.tsx:7]: delete if is empty

File: src/components/CryptCard.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+const CryptCard = (Props: Props) => {
+  let percentageChange = (percentNumber: number) => {
+    let roundNumber = Math.round((percentNumber + Number.EPSILON) * 100) / 100;
```
Comment (tomasdematos) [inline src/components/CryptCard.tsx:11]: use conts

File: src/components/CryptCard.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+    if (roundNumber > 0) {
```
Comment (tomasdematos) [inline src/components/CryptCard.tsx:13]: redundant components:

```
const negative = roundNumber < 0

return (
<View style={styles.percentChangeText}>
          <GainArrow width={16} height={16} fill={negative ?  "red" : "green"} />
          <Text style={styles.textGain}>{(negative ? -roundNumber : roundNumber) + '%'}</Text>
        </View>
)
```

Comment (tomasdematos) [reply src/components/CryptCard.tsx:13]: also use styled components, a styled component can get props, so you can validate the color of the text inside the styles

File: src/components/CryptCard.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+          )}
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default CryptCard;
+
+const styles = StyleSheet.create({
```
Comment (tomasdematos) [inline src/components/CryptCard.tsx:63]: styles should be stored in a separate file, the most common way to do this is with this format:

```
src/
   Components/
      AddButton/
          index.tsx <---- this file 
          styles.ts <---- move the styles here
```

File: src/components/CryptCards.tsx [hunk 5]
```diff
@@ -1,17 +1,15 @@
...
 
 const CryptCards = (props: Props) => {
   return (
-    <FlatList
-      data={crypto}
-      renderItem={({item: crypt}) => (
-        <CryptCard {...crypt}></CryptCard>
-      )}/>
+    <View>
+      <CryptCard></CryptCard>
```
Comment (tomasdematos) [inline src/components/CryptCards.tsx:11]: you can use:       <CryptCard />
if it doens't have a child

**Pull request closed without merging**